packages/table-ui/package.json

```diff
@@ -47,7 +52,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "slate": "^0.62.0",
-    "slate-react": "^0.62.0"
```

Comment:
needed.

Reply:
fixed

Reply:
@roosnic1 fixed
Is it normal to have these four packages in peerDependencies and devDependencies?

Reply:
fixed 👍